### ISSUE TITLE
[2018.06.14] AutoAugment: Learning Augmentation Policies from Data

### ISSUE THREAD

**Issue by unizard** [unizard/AwesomeArxiv#198]
Institute: Google Brain
URL: https://arxiv.org/pdf/1805.09501.pdf
Keyword: Data Augmentation, AutoML, ReinforceLearning
Interest: 5

**Summary**
In this paper, we take a closer look at data augmentation for images, and describe a simple procedure called AutoAugment to search for improved data augmentation policies. On ImageNet, we attain a Top-1 accuracy of 83.54%. On CIFAR-10, we achieve an error rate of 1.48%, which is 0.65% better than the previous state-of-the-art. On reduced data settings, AutoAugment performs comparably to semi-supervised methods without using any unlabeled examples.

**Comment by unizard** [comment]
Related Work
- [Automatic Dataset Augmentation](https://arxiv.org/pdf/1708.08201v1.pdf), CoRR 2017. MSRA
    - https://auto-da.github.io/

**Comment by unizard** [comment]
## Results
- Validation set Top-1 / Top-5 error rates (%) on ImageNet. Lower is better.
    ![image](https://user-images.githubusercontent.com/11758940/41388094-4379829c-6fc5-11e8-8074-dd13faccdc9e.png)

- Test set Top-1 error rates (%) on FGVC datasets. AutoAugment results use the policy found on ImageNet to train an Inception v4 from scratch.
    ![image](https://user-images.githubusercontent.com/11758940/41388224-ef0029fe-6fc5-11e8-8256-dc805ecf4cde.png)

**Comment by unizard** [comment]
-  List of all image transformations, AutoAugment policy found on reduced ImageNet.

| 1 | 2 |
|:--:|:--:|
| ![image](https://user-images.githubusercontent.com/11758940/41388470-08455c6c-6fc7-11e8-9a19-b2f836104697.png) | ![image](https://user-images.githubusercontent.com/11758940/41388422-cf4d4488-6fc6-11e8-8c4a-8ae382b18803.png)|
